include/chewingio.h

```diff
@@ -545,7 +545,7 @@ CHEWING_API int chewing_get_phoneSeqLen(const ChewingContext *ctx);
 CHEWING_API void chewing_set_logger(ChewingContext *ctx,
                                     void (*logger) (void *data, int level, const char *fmt, ...), void *data);
 
-CHEWING_API void chewing_set_logger2(ChewingContext *ctx,
+CHEWING_API void chewing_set_logger_without_varglist(ChewingContext *ctx,
```

Comment:
The suffix `without_varglist` is too long. I would expected to revise the use of API instead.